public/js/action.js

```diff
@@ -20,11 +20,84 @@ export class Action {
                     this.scene.openReading();
                 }
             },
+
             // AOG Education Geography Commands
+            LOAD_STATE_MAP: (data) => {
```

Comment:
We can discuss this in stand up but I think it might be best to create a function with this code inside of it then call the function in one line here. This is because we will perhaps have a lot of actions and it may start to get cluttered. You can create your function with this same code in your scene.js file (not sure how it will work if we create functions inside the action.js file)